### DISCUSSION TITLE
Input Text Float Label issue with v.17.0.0-RC.1

### DISCUSSION THREAD

**Discussion by 4gDad** [primefaces/community#581] [PrimeNG] [answered]
I have not been able to get float labels to work in either of the pre-release version of v17.   I even tested this simple code, in the PrimeNG playground and was able to duplicate issue.  

```
<span class="p-float-label">
    <input pInputText id="username" [(ngModel)]="value" />
    <label htmlFor="username">Username</label>
</span>
```

![image](https://github.com/primefaces/community/assets/32912937/0d513fcf-81d0-4e69-9ab7-eda517c652bd)

**Comment by mehmetcetin01140** [comment, marked as answer]
Hi,

It works in this example, you can examine it.
https://stackblitz.com/edit/mp818s?file=src%2Fapp%2Fdemo%2Finput-text-icons-demo.ts

**Reply by 4gDad** [reply]
Strange because when I create the same thing in the Angular Playground I get a different result.  
https://stackblitz.com/edit/stackblitz-starters-6pa9zb?file=src%2Fmain.ts

**Reply by ThoSap** [reply]
You are missing the import
```scss
@import "primeng/resources/primeng.css";
```
in file `global_styles.css`.
I just tried it with your Stackblitz @4gDad and this import fixed it.

See https://primeng.org/installation#styles

**Reply by ThoSap** [reply]
@4gDad did my solution help you?

**Reply by ghost** [reply]
> ```css-scss
> @import "primeng/resources/primeng.css";
> ```

This helped me :) thank you very much.
